weave-js/src/components/FancyPage/useProjectSidebar.ts

```diff
@@ -161,7 +161,11 @@ export const useProjectSidebar = (
           {
             type: 'button' as const,
             name: 'Evals',
-            slug: 'weave/evaluations',
+            slug: 'weave/evaluations/evals',
```

Comment:
`/evaluations/evals` seems odd to me I would try to be specific like `evaluations/runs` or `evaluations/batch`.